src/Agents.jl

```diff
@@ -47,6 +48,28 @@ function __init__()
     @require Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4" begin
         include("visualization/plot-recipes.jl")
     end
```

Comment:
Wait, what's this doing here? Guess we can remove it, right @Libbum ?

Reply:
No, that needs to stay or the docs will die. It's fine as-is.